national_rail/extract.py

```diff
@@ -0,0 +1,27 @@
+"""Gets the information from the national rail api"""
+
+from os import environ as ENV
+
+from requests import get
+from dotenv import load_dotenv
+
+
+def get_data_from_api() -> None:
+    """Gets the data from the national rail api and saves it to a file"""
+    load_dotenv()
```

Comment:
now why is this here? I think it should be in the if name block

Reply:
Yes that's possible but if this function is called form another file, they wouldn't need to put in load_dotenv()